src/scenes/Authentication/ResetPassword/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,74 @@
+import { Card, CardContent } from '@material-ui/core';
+import { Decorator, Mutator } from 'final-form';
+import React from 'react';
+import { Form, FormProps } from 'react-final-form';
+import { ResetPasswordInput } from '../../../api';
+import {
+  blurOnSubmit,
+  focusLastActiveFieldOnSubmitError,
+  SubmitButton,
+  SubmitError,
+  TextField,
+} from '../../../components/form';
+
+export type ResetPasswordFormProps = Pick<
+  FormProps<ResetPasswordInput>,
+  'onSubmit' | 'initialValues'
+> & { className?: string };
+
+export const ResetPasswordForm = ({
```

Comment:
Same changes here as other form. We'll also want a second password field to validate new password. I'm fine if you want to skip this for now since that validation is a little tricky.